docs/blog-post.md

```diff
@@ -0,0 +1,286 @@
+<h1><b>How to make akka serialization bulletproof</b></h1>
+
+![bulletproof shield](https://images.unsplash.com/photo-1561156772-a44477f220a5?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1974&q=80)
+<i>bulletproof shield from [Unsplash](https://images.unsplash.com/photo-1561156772-a44477f220a5) by [NIFTYART](https://unsplash.com/@niftyartofficial1_)</i>
+
+Every message leaving a JVM boundary in Akka needs to be serialized first. However, the existing solutions for serialization in Scala leave a lot of room for runtime failures that are NOT reported in compile time. We’re glad to introduce [Akka Serialization Helper](https://github.com/VirtusLab/akka-serialization-helper), a toolkit including a Circe-based, runtime-safe serializer and a set of Scala compiler plugins to counteract the common caveats when working with Akka serialization.
+
+![logo_ash_horizontal@4x](https://user-images.githubusercontent.com/25779550/135059025-4cfade5b-bfcb-47e8-872f-8a3d78ce0c25.png)
+
+While Akka is generally a great tool to work with serialization, there are few specific situations, where things might go wrong and cause unexpected errors. Specifically, things that make standard akka serialization prone to errors are:
+
+1. [Missing serialization binding](#1-missing-serialization-binding)
+2. [Incompatibility of persistent data](#2-incompatibility-of-persistent-data)
+3. [Jackson Akka Serializer drawbacks](#3-jackson-akka-serializer-drawbacks)
+4. [Missing Codec registration](#4-missing-codec-registration)
+
+All of these problems have one thing in common - these are bugs in application code (programmer's oversights) that are not detected in compile-time, but can easily break your app in runtime. Fortunately - you can get rid of all these issues by catching them in compilation, because Akka Serialization Helper (shortly: ASH) comes to the rescue!
+
+## How to install ASH in your project
+
+Add the following line to `plugins.sbt` (take `Version` from the above maven badge or [GitHub Releases](https://github.com/VirtusLab/akka-serialization-helper/releases)):
+
+```scala
+addSbtPlugin("org.virtuslab.ash" % "sbt-akka-serialization-helper" % Version)
+```
+
+and enable the sbt plugin in the target project:
+```scala
+lazy val app = (project in file("app"))
+  .enablePlugins(AkkaSerializationHelperPlugin)
+```
+
+## <i>Ok, but why would I ever need to use that?</i>
```

Comment:
Hmmm this paragraph seems somewhat redundant/out of place &mdash; you've introduced the general rationale for using ASH above already.

Reply:
I changed it's name to "Akka Serialization Helper usage". The paragraph itself is needed in order to divide it visually from the previous "How to install ASH in your project" paragraph. :)